docs/demultiplexing.md

```diff
@@ -160,6 +160,52 @@ Example configuration file for pipeline with `gather_from_node` specified before
 }
 ```
 
+## Dynamic batch handling with demultiplexing
+
+Demutliplexing feature enables handling requests with dynamic batch size. For workloads that involve frequent batch size change this is recommended way to handle such requests.
+In order to use it pipeline config has to have additional field demultiply_count. To leverage this feature requests, and responses have to have additional dimension of inputs/outputs.
```

Comment:
You can use dynamic batching feature via requests demuliplexing by configuring a pipeline including a single model and an extra field 'demultiply_count: 0'. To leverage this feature, input data requires additional, first dimension, representing the batch size. It should be added to the original model shape with batch size 1. The response will include combined predictions from the split batches. That also adds extra first dimension to the model output.